cmd/build-iso.go

```diff
@@ -139,6 +140,7 @@ func NewBuildISO(root *cobra.Command, addCheckRoot bool) *cobra.Command {
 	c.Flags().String("overlay-uefi", "", "Path of the overlayed uefi data")
 	c.Flags().String("overlay-iso", "", "Path of the overlayed iso data")
 	c.Flags().String("label", "", "Label of the ISO volume")
+	c.Flags().String("extra-cmdline", constants.ISODefaultExtraCmdline, "Extra kernel cmdline")
```

Comment:
I'd keep this default empty and include the default in `pkg/config/config.go` like other flags. This way the default is applied either you provide a command line flag or you define it in the yaml manifest file.

Reply:
Good point,  I will update :+1: